lib/recurly/purchase.php

```diff
@@ -10,6 +10,9 @@
  * @property string[] $coupon_codes An array of coupon codes to apply to the purchase
  * @property Recurly_Subscription[] $subscriptions An array of subscriptions to apply to the purchase
  * @property Recurly_GiftCard $gift_card A gift card to apply to the purchase
+ * @property string $customer_notes Optional notes field. This will default to the Customer Notes text specified on the Invoice Settings page in your Recurly admin. Specify custom notes with this tag to add Customer Notes. Custom notes made on an invoice for a one time charge will not carry over to subsequent invoices.
```

Comment:
Can we drop the "Specify custom notes with this tag to add Customer Notes." bit?

Reply:
yes